src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadSubmissionResource.java

```diff
@@ -178,6 +178,8 @@ public FileUploadSubmissionResource(FileUploadSubmissionRepository fileUploadSub
             fileUploadSubmissions = fileUploadSubmissionService.getFileUploadSubmissions(exerciseId, submittedOnly);
         }
 
+        fileUploadSubmissions.forEach(fileUploadSubmissionService::hideDetails);
```

Comment:
Some existing (as well as the newly added) integration tests fail when adding this line:
```
org.springframework.orm.jpa.JpaSystemException:
A collection with cascade="all-delete-orphan" was no longer referenced by the owning entity instance: 
de.tum.in.www1.artemis.domain.Exercise.studentParticipations;

nested exception is org.hibernate.HibernateException:
A collection with cascade="all-delete-orphan" was no longer referenced by the owning entity instance: 
de.tum.in.www1.artemis.domain.Exercise.studentParticipations
```


We do the same thing for text exercises which works just fine. I cannot spot the difference in the setup.
I guess this might be related with the `@Transactional` Annotation used in the `FileUploadSubmissionService`, but also we have the same setup for text.
@krusche @maxr96 Can one of you help me look into this?

Reply:
Wait why are you doing this two times?

Reply:
Oh apparently I messed up when doing a rebase on develop.